### PR TITLE
Add pancakeswap_v2_ethereum_trades

### PR DESCRIPTION
**Pull request by chef-seaweed**
Brief comments on the purpose of your changes:

**For adding trades table for Pancakeswap on Ethereum**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
* [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: seeds/pancakeswap/ethereum/pancakeswap_v2_ethereum_trades_samples.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
+ethereum,pancakeswap,2,2023-01-09 0:00,2023-01-09 3:22,USDC,WETH,USDC-WETH,39.768234,0.03053786191,39768234,30537861913928636,39.75666144,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0x952454d0915cc41f30dc26f76add170124e3efb4,,0x2e8135be71230c6b1b4045696d41c09db0414226,0xc1f7f412ac0636e914b2e55e21ddbb79f313be2d7ca4c5e084db223787c52441,0x952454d0915cc41f30dc26f76add170124e3efb4,0xeff92a263d31888d860bd50809a8d171709b7b1c,,325
+ethereum,pancakeswap,2,2022-12-01 0:00,2022-12-01 11:34,WETH,USDT,USDT-WETH,7.607205635,9627.386202,7.60721E+18,9627386202,9749.851175,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0xdac17f958d2ee523a2206206994597c13d831ec7,0xa8e7437833f4c2f4f36f8e4559db91bea951bfa9,,0x17c1ae82d99379240059940093762c5e4539aba5,0x70cb37c8e8a566b850d1475606b3c4b64adf51cca9bb9e32645344b93c99c8f9,0x89eed9db3903e8c17bceb5f03d7a10cde8733c62,0xa8e7437833f4c2f4f36f8e4559db91bea951bfa9,,17
```
Comment (Hosuke) [inline seeds/pancakeswap/ethereum/pancakeswap_v2_ethereum_trades_samples.csv:4]: Hi @chef-seaweed, we can move the seed to [dex_trades_seed.csv](https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv) for consistency.

Comment (chef-seaweed) [reply seeds/pancakeswap/ethereum/pancakeswap_v2_ethereum_trades_samples.csv]: Hi @Hosuke,thanks for the advice.  I've moved the sample data to [dex_trades_seed.csv](https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv) and removed [pancakeswap_v2_ethereum_trades_samples.csv](https://github.com/duneanalytics/spellbook/pull/2806/commits/21f90a855e472114820b80c2a509e19cac744a6d#diff-5910b5426129647e37d4628008bac9a15dd61f5daf01381e05808f32d739d20d). Also would like to what else need to be done to make pancakeswap_v2_ethereum_trades data included in dex.trades? Thank you.

File: seeds/dex/trades/dex_trades_seed.csv [hunk 2]
```diff
@@ -259,10 +255,8 @@ optimism,zipswap,1,2022-02-05,0x111251020222b3efbd71693d4d5ef3f3e3ff9598f0231528
...
-fantom,beethoven_x,1,2023-02-01,0xb353ea3753bd43f79452a99a5f0b4ba79848eb535406006b2a325e2610cc20c7,10,0xc3bf643799237588b7a6b407b3fc028dd4e037d2,10.79068245,0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83,10.9191819
-fantom,beethoven_x,1,2023-02-01,0x406e3b0bfa2b7289eb7a9c37a87f55d1c1f7b5dc1c90cf638ccf6bc06b73f4f0,43,0xc3bf643799237588b7a6b407b3fc028dd4e037d2,154.8200719,0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83,156.6637268
-fantom,beethoven_x,1,2023-02-01,0x9ed054600393f6ddfd5e9e271ceacc419ea1ea5c5069032f3bccf6e0bbd1057c,9,0x8d11ec38a3eb5e956b052f67da8bdc9bef8abf3e,4.419601846,0x04068da6c83afcfa0e13ba15a6696662335d5b75,4.417833
-fantom,beethoven_x,1,2023-02-01,0x3ad542dcee62446ad4cea36a4606c1dfecdc654d83113509ca43c958737b97b9,44,0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83,1999.4,0x04068da6c83afcfa0e13ba15a6696662335d5b75,1042.833542
-fantom,beethoven_x,1,2023-02-01,0xf73a2c4c02eb03d263d39f5b640229bb9b23f8242a5389b37a5c8aedc428b17e,56,0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83,84.55916959,0xc5e2b037d30a390e62180970b3aa4e91868764cd,550.564336
```
Comment (Hosuke) [inline seeds/dex/trades/dex_trades_seed.csv:268]: It seems dex_trades_seed file damaged mistakenly during the update, I will recover the damaged seed in the next commit.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @chef-seaweed 

[Sample pancake v2 ethereum trades](https://dune.com/queries/2050282?d=1)

**Comment by chef-seaweed** [comment]
Thank you @Hosuke, hi @jeff-dude I've added some minor updates to pancakeswap_trades_schema.yml to make it more like uniswap_trades_schema.yml, thanks

**Comment by jeff-dude** [comment]
pushed two updates:
- switched incremental filter to be where clause on base source table
- added back ethereum address for `STG` token in prices, it should be okay to have the `id` repeated across different blockchains, as long as address is as expected. [bnb](https://bscscan.com/token/0xb0d502e938ed5f4df2e681fe6e419ff29631d62b), [ethereum](https://etherscan.io/token/0xaf5191b0de278c7286d6c7cc6ab6bb8a73ba2cd6). we try to call that out in the logs:
> (Not uncommon! share block explorer link to confirm contract)

**Pull request merged**